inc/namespace.php

```diff
@@ -205,6 +214,15 @@ function replace_attached_file( $file, int $attachment_id ) : string {
 	return $metadata['file'] ?? '';
 }
 
+function replace_attachment_url( string $url, int $attachment_id ) : string {
+	$attachment = get_post( $attachment_id );
+	if ( ! is_amf_asset( $attachment ) ) {
```

Comment:
Just noting that, in case `$attachment`is `null`, this will fail as `is_amf_asset` expects a `WP_Post`.
Question is if we can run into this edge case...? I doubt we can, as the filter should only ever get executed (by WordPress) for a valid attachment, right?

Reply:
In theory but you're right - it'll be more robust to make `is_amf_asset()` handle a null value too.